core/controllers/topics_and_skills_dashboard.py

```diff
@@ -166,6 +166,36 @@ def get(self):
         self.render_json(self.values)
 
 
+class UnassignSkillDataHandler(base.BaseHandler):
+    """Provides data for Unassign skill modal."""
```

Comment:
This is named wrongly. It shouldn't have reference to anything in the frontend -- this is a backend handler which receives requests and responds to them.

In this case, the handler is providing a list of topics that the skill is assigned to. So it should be something like SkillAssignmentsHandler and the GET should be "retrieving information about which topics contain the given skill". Note how this handler can then be reused for other things.

Reply:
Hmm yeah, will keep that in mind. Done.